production/deploy/azure/terraform/main.tf

```diff
@@ -1,3 +1,17 @@
+# Copyright 2024 Google LLC
```

Comment:
For new files we use `# Portions Copyright (c) Microsoft Corporation`

https://github.com/KenGordon/bidding-auction-servers/blob/dev/production/packaging/azure/buyer_frontend_service/bin/init_server_basic